playbooks/roles/setup-caasp-workers/files/caasp_cp_node_set_subvolumes.sh

```diff
@@ -10,7 +10,7 @@ systemctl stop docker
 mounted_snapshot=$(mount | grep snapshot | gawk  'match($6, /ro.*@\/.snapshots\/(.*)\/snapshot/ , arr1 ) { print arr1[1] }')
 
 if [ -n "$mounted_snapshot" ]; then
-  btrfs property set -ts /.snapshots/$mounted_snapshot/snapshot ro false
+  btrfs property set -ts /.snapshots/"$mounted_snapshot"/snapshot ro false
```

Comment:
(nit) Should this be `"${mounted_snapshot}"`?

Reply:
:+1: